sdk/servicebus/pom.service.xml

```diff
@@ -0,0 +1,30 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.azure</groupId>
+  <artifactId>azure-servicebus-service</artifactId>
+  <packaging>pom</packaging>
+  <version>1.0.0</version><!-- Need not change for every release-->
+
+  <name>Microsoft Azure Service Bus SDK Service</name>
+  <description>Aggregator POM for Azure Service Bus SDK</description>
+  <url>https://github.com/Azure/azure-sdk-for-java</url>
+
+  <distributionManagement>
```

Comment:
Do you know if this is needed? It would be ideal if we make this simple aggregate pom as small and simple as possible as it will be getting copied and pasted everywhere.

Reply:
I'll see how small I can get it before things blow up :)

Reply:
So the group id, artifact name, version and packaging is all required, but everything else can be trimmed out.